src/coreclr/vm/jitinterface.cpp

```diff
@@ -1636,6 +1645,60 @@ uint32_t CEEInfo::getThreadLocalFieldInfo (CORINFO_FIELD_HANDLE  field, bool isG
     return typeIndex;
 }
 
+#ifndef _MSC_VER
```

Comment:
This is OS-specific code, so it should use OS ifdef.

For example - if we were to use clang on Windows, this ifdef would not be correct.